partials/pagination.php

```diff
@@ -1,24 +1,39 @@
 <?php
-if( get_next_posts_link() || get_previous_posts_link() ) {
-  $previous = get_previous_posts_link('Newer');
-  $next = get_next_posts_link('Older');
+if (is_single()) {
+  $prev_post = get_previous_post();
+  $next_post = get_next_post();
+  $older_link = false;
+  $newer_link = false;
+
+  if (!empty($prev_post)) {
+    $older_link = get_permalink($prev_post);
+  }
+  if (!empty($next_post)) {
+    $newer_link = get_permalink($next_post);
+  }
+} else {
+  $newer_link = get_previous_posts_page_link();
```

Comment:
For some reason these are returning true on the first and last page of pagination so displaying dead links. This could be a new problem in the starterpack as looks the same as the boiler code